### ISSUE TITLE
dirac-admin-add-pilot parameters mismatch with pilotCommand call

### ISSUE THREAD

**Issue by andresailer** [DIRACGrid/DIRAC#7622]
Compare
https://github.com/DIRACGrid/DIRAC/blob/baf7bb530b724b53d97fa74fccba19716447a9da/src/DIRAC/WorkloadManagementSystem/scripts/dirac_admin_add_pilot.py#L71-L76

with

https://github.com/DIRACGrid/Pilot/blob/c9bfc413ce862c66391892e05fda63f5dc8a96bd/Pilot/
```python
pilotCommands.py#L655-L661
        checkCmd = "dirac-admin-add-pilot %s %s %s %s --status=Running %s -d" % (
            self.pp.pilotReference,
            self.pp.wnVO,
            self.pp.flavour,
            self.pilotStamp,
            " ".join(self.cfg),
        )
```

And pilots having the message
```
2024-05-20T06:36:57.837860Z INFO [RegisterPilot] Command RegisterPilot instantiated from pilotCommands
2024-05-20T06:36:57.837960Z INFO [RegisterPilot] Executing command dirac-admin-add-pilot htcondorce://grid-htcondorce1.desy.de/7080645.0 ilc HTCondorCE c8a007f66fa6d26441488f400362af85 --status=Running --cfg pilot.cfg -d
Error when parsing command line arguments: not all mandatory arguments are defined.
```

So ownergroup/ownerdn should be VO or vice versa?

**Comment by fstagni** [comment]
TLDR: it effectively only works with DIRAC v9. The error shown is not dangerous, so ignore it.


In integration branch of DIRAC `dirac-admin-add-pilot` has the following switches:

```python
    Script.registerSwitches(params.switches)
    Script.registerArgument("pilotRef: pilot reference")
    Script.registerArgument("VO: VO, or pilot owner group")
    Script.registerArgument("gridType: grid type")
    Script.registerArgument("pilotStamp: DIRAC pilot stamp")
```
So, basically there is an interface change because of the [many DB changes](https://github.com/DIRACGrid/DIRAC/wiki/DIRAC-9.0#remove-of-diracsetup-and-ownerdn-in-wms-and-transformations-plus-several-other-simplifications-and-changes). In the end I gave up on trying to make this work for every release (https://github.com/DIRACGrid/Pilot/pull/234/files, https://github.com/DIRACGrid/Pilot/pull/226).

**Comment by andresailer** [comment]
Thanks for the explanation @fstagni !
